### PR TITLE
SAK-46392 Trinity portal implementation

### PR DESCRIPTION
**Pull request by ern**
https://sakaiproject.atlassian.net/browse/SAK-46392

All the commits that that make the new portal

Co-authored-by: Kunal Jaykam <kunaljaykam@gmail.com>
Co-authored-by: Shawn Foster <shawn.foster@uwo.ca>
Co-authored-by: Michael Greene <3632822+profmikegreene@users.noreply.github.com>
Co-authored-by: Markus Stetschnig <dev@stetschom.at>
Co-authored-by: Sandro Kopp <sandrokopp90@gmail.com>
Co-authored-by: Jürgen Schmaulzer <juergen.schmautzer@htl-wolfsberg.at>
Co-authored-by: Adrian Fish <adrian.r.fish@gmail.com>

### CODE REVIEW

File: assignment/api/src/java/org/sakaiproject/assignment/api/AssignmentConstants.java [hunk 1]
```diff
@@ -351,17 +351,17 @@ public enum IMSGradingProgress {
     public static final String GRADE_SUBMISSION_FEEDBACK_ATTACHMENT = "grade_submission_feedback_attachment";
 
     // submission level of resubmit due time
-    public static final String ALLOW_RESUBMIT_CLOSEYEAR = "allow_resubmit_closeYear";
-    public static final String ALLOW_RESUBMIT_CLOSEMONTH = "allow_resubmit_closeMonth";
-    public static final String ALLOW_RESUBMIT_CLOSEDAY = "allow_resubmit_closeDay";
-    public static final String ALLOW_RESUBMIT_CLOSEHOUR = "allow_resubmit_closeHour";
-    public static final String ALLOW_RESUBMIT_CLOSEMIN = "allow_resubmit_closeMin";
+    public static final String ALLOW_RESUBMIT_CLOSEYEAR = "allow_resubmit_closeyear";
```
Comment (jesusmmp) [inline assignment/api/src/java/org/sakaiproject/assignment/api/AssignmentConstants.java:354]: Avoid unnecesary changes on this file.

File: assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java [hunk 2]
```diff
@@ -1049,16 +1049,16 @@ public class AssignmentAction extends PagedResourceActionII {
...
-    private static final String ALLPURPOSE_RELEASE_MONTH = "allPurpose_releaseMonth";
-    private static final String ALLPURPOSE_RELEASE_DAY = "allPurpose_releaseDay";
-    private static final String ALLPURPOSE_RELEASE_HOUR = "allPurpose_releaseHour";
-    private static final String ALLPURPOSE_RELEASE_MIN = "allPurpose_releaseMin";
-    private static final String ALLPURPOSE_RETRACT_YEAR = "allPurpose_retractYear";
-    private static final String ALLPURPOSE_RETRACT_MONTH = "allPurpose_retractMonth";
-    private static final String ALLPURPOSE_RETRACT_DAY = "allPurpose_retractDay";
-    private static final String ALLPURPOSE_RETRACT_HOUR = "allPurpose_retractHour";
-    private static final String ALLPURPOSE_RETRACT_MIN = "allPurpose_retractMin";
+    private static final String ALLPURPOSE_RELEASE_YEAR = "allPurpose_releaseyear";
```
Comment (jesusmmp) [inline assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java:1052]: Same here, avoid unnecesary changes

File: assignment/tool/src/webapp/js/assignments.js [hunk 3]
```diff
@@ -982,9 +982,11 @@ ASN.disableTimesheetSetupSection = function()
     ASN.toggleAutoAnnounceEstimate(false);
 }
 
+//$(document).noConflict().ready(() => {
```
Comment (jesusmmp) [inline assignment/tool/src/webapp/js/assignments.js:985]: Remove comment

File: library/src/skins/default/src/sass/base/_defaults.scss [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+    }
+    &.btn-primary{
+      @include sakai_primary_button();
+    }
+    text-decoration: none;  // TODO
+  }
+  &.url{
+    @media #{$phone}{
+        display: block;
+      max-width: 100%;
```
Comment (jesusmmp) [inline library/src/skins/default/src/sass/base/_defaults.scss:92]: Spacing

File: library/src/skins/default/src/sass/modules/tool/rubrics/_rubrics.scss [hunk 5]
```diff
@@ -763,3 +763,39 @@ dl {
...
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-bottom: 19px solid #f8f9fa;
+    width: 0;
+    height: 0;
+    -webkit-filter: drop-shadow(0 -2px 2px #AAA);
+  }
+
+.arrow-0 {
+	position: absolute;
```
Comment (jesusmmp) [inline library/src/skins/default/src/sass/modules/tool/rubrics/_rubrics.scss:780]: Spacing

**Review by adrianfish — APPROVED** [review]

**Pull request merged**